crates/sui-transaction-builder/src/lib.rs

```diff
@@ -406,13 +406,16 @@ impl<Mode: ExecutionMode> TransactionBuilder<Mode> {
             package.linkage_table,
         )?;
 
+        let max_binary_format_version =
+            ProtocolConfig::get_for_min_version().move_binary_format_version();
```

Comment:
I think we are going to need to eventually fetch the protocol version from the validators for the current epoch, if we want to build a transaction for the current epoch. If this is running on a fullnode, the fullnode's current epoch is good enough.

Reply:
In this case, though, this is isn't used to construct a transaction for a specific protocol version, but rather to figure out the types of Move arguments for an already existing module so that (presumably) the Move call can be constructed on the correctly. In this case, it should be OK to use the max version I think? Will add a comment along these lines, but please let me know if I got this wrong.